target/linux/mvebu/image/cortex-a9.mk

```diff
@@ -183,4 +183,21 @@ define Device/cznic_turris-omnia
 endef
 TARGET_DEVICES += cznic_turris-omnia
 
+define Device/wd_mcex2u
+  $(Device/NAND-128K)
+  DEVICE_TITLE := WD My Cloud EX2 Ultra (BVBZ/Ranger Peak)
+  DEVICE_PACKAGES += wtmcu -uboot-envtools -dnsmasq -ppp -ppp-mod-pppoe \
+    block-mount e2fsprogs partx-utils mkf2fs f2fsck \
+    kmod-dm kmod-md-mod kmod-fs-vfat kmod-fs-ext4 kmod-fs-xfs \
+    kmod-usb-ledtrig-usbport kmod-usb2 kmod-usb3 kmod-usb-storage kmod-usb-xhci
+  DEVICE_DTS := armada-385-wd-mcex2u
+  KERNEL_SIZE := 5120k
+  IMAGES += image.cfs-factory.img
+  IMAGE/image.cfs-factory.img := append-ubi
+  IMAGES += uImage-factory.img
+  IMAGE/uImage-factory.img := append-kernel
+  SUPPORTED_DEVICES += armada-385-wd-mcex2u wd,mcex2u
```

Comment:
You can remove this line. `wd,mcex2u` is set in Default recipe and previously we used dts name as supported devices string, but since Your board was not supported before, there is no need to add it.